packages/dashboard/src/components/internalDashboard/index.tsx

```diff
@@ -54,13 +67,19 @@ const selectedRect = (selection: Selection | undefined): Rect | undefined => {
 };
 import SidePanel from '../side-panel';
 
-const InternalDashboard = () => {
+type InternalDashboardProps = {
+  // merge(this.messageOverrides, DefaultDashboardMessages)
```

Comment:
Not sure if leaving this in is intentional?

Reply:
Nope missed this!